### PR TITLE
Update db-xrefs.yaml

### PR DESCRIPTION
**Pull request by zhilianghu**
Added xref for (1) animal QTLdb; (2) animal CorrDB

### CODE REVIEW

File: metadata/db-xrefs.yaml [hunk 1]
```diff
@@ -2887,3 +2887,22 @@
   entity_types:
     - type_name: entity
       type_id: BET:0000000
+- database: Animal QTLdb
+  name: QTLdb
+  generic_urls: https://www.animalgenome.org/QTLdb
+  description: A database with curated publicly available trait mapping data in livestock animal species
+  entity_types: 
+    - type_name: entity
+    - url_syntax: https://www.animalgenome.org/QTLdb/q?id=[QTL_ID]
```
Comment (kltm) [inline metadata/db-xrefs.yaml:2896]: This should probably be `[example_id]`

File: metadata/db-xrefs.yaml [hunk 2]
```diff
@@ -2887,3 +2887,22 @@
...
+    - id_syntax: \d+
+    - example_id: 23155
```
Comment (kltm) [inline metadata/db-xrefs.yaml:2898]: This would need to be a CURIE

File: metadata/db-xrefs.yaml [hunk 3]
```diff
@@ -2887,3 +2887,22 @@
...
+    - example_url: https://www.animalgenome.org/QTLdb/q?id=23155
+- database: Animal CorrDB
+  name: CorrDB
+  description: A database with curated publicly available trait correlation and inheritability data in livestock animal species
+  entity_types: 
+    - type_name: entity
+    - url_syntax: https://www.animalgenome.org/CorrDB/q/?id=[example_id]
+    - id_syntax: \d+
+    - example_id: 37232
```
Comment (kltm) [inline metadata/db-xrefs.yaml:2907]: This would need to be a CURIE

File: metadata/db-xrefs.yaml [hunk 4]
```diff
@@ -2887,3 +2887,22 @@
   entity_types:
     - type_name: entity
       type_id: BET:0000000
+- database: Animal QTLdb
```
Comment (kltm) [inline metadata/db-xrefs.yaml:2890]: This needs to be the CURIE namespace.

File: metadata/db-xrefs.yaml [hunk 5]
```diff
@@ -2887,3 +2887,22 @@
...
+  name: QTLdb
+  generic_urls: https://www.animalgenome.org/QTLdb
+  description: A database with curated publicly available trait mapping data in livestock animal species
+  entity_types: 
+    - type_name: entity
+    - url_syntax: https://www.animalgenome.org/QTLdb/q?id=[QTL_ID]
+    - id_syntax: \d+
+    - example_id: 23155
+    - example_url: https://www.animalgenome.org/QTLdb/q?id=23155
+- database: Animal CorrDB
```
Comment (kltm) [inline metadata/db-xrefs.yaml:2900]: This needs to be the CURIE namespace.

**Review by kltm — CHANGES_REQUESTED** [review]

File: metadata/db-xrefs.yaml (hunk 3 above)
Comment (zhilianghu) [reply metadata/db-xrefs.yaml:2907]: Yes; Changed to [Animal_QTLdb]

File: metadata/db-xrefs.yaml (hunk 1 above)
Comment (zhilianghu) [reply metadata/db-xrefs.yaml:2896]: Changed

File: metadata/db-xrefs.yaml (hunk 5 above)
Comment (zhilianghu) [reply metadata/db-xrefs.yaml:2900]: Changed to [Animal_CorrDB]

**Comment by zhilianghu** [comment]
Did another push.

**Comment by kltm** [comment]
Your PR is still failing:
```
0.16s$ kwalify -E -f metadata/db-xrefs.schema.yaml metadata/db-xrefs.yaml 2>&1 | tee out.log && grep 'INVALID\|ERROR' out.log; test $? -ne 0
metadata/db-xrefs.yaml#0: INVALID
metadata/db-xrefs.yaml:2892:3 [/264/generic_urls] 'https://www.animalgenome.org/QTLdb': not a sequence.
metadata/db-xrefs.yaml:2895:7 [/264/entity_types/0] key 'type_id:' is required.
metadata/db-xrefs.yaml:2896:7 [/264/entity_types/1] key 'type_name:' is required.
metadata/db-xrefs.yaml:2896:7 [/264/entity_types/1] key 'type_id:' is required.
metadata/db-xrefs.yaml:2897:7 [/264/entity_types/2] key 'type_id:' is required.
metadata/db-xrefs.yaml:2897:7 [/264/entity_types/2] key 'type_name:' is required.
metadata/db-xrefs.yaml:2898:7 [/264/entity_types/3] key 'type_name:' is required.
metadata/db-xrefs.yaml:2898:7 [/264/entity_types/3] key 'type_id:' is required.
metadata/db-xrefs.yaml:2898:7 [/264/entity_types/3/example_id] '23155': not a string.
metadata/db-xrefs.yaml:2899:7 [/264/entity_types/4] key 'type_id:' is required.
metadata/db-xrefs.yaml:2899:7 [/264/entity_types/4] key 'type_name:' is required.
metadata/db-xrefs.yaml:2900:3 [/265] key 'generic_urls:' is required.
metadata/db-xrefs.yaml:2904:7 [/265/entity_types/0] key 'type_id:' is required.
metadata/db-xrefs.yaml:2905:7 [/265/entity_types/1] key 'type_name:' is required.
metadata/db-xrefs.yaml:2905:7 [/265/entity_types/1] key 'type_id:' is required.
metadata/db-xrefs.yaml:2906:7 [/265/entity_types/2] key 'type_name:' is required.
metadata/db-xrefs.yaml:2906:7 [/265/entity_types/2] key 'type_id:' is required.
metadata/db-xrefs.yaml:2907:7 [/265/entity_types/3] key 'type_name:' is required.
metadata/db-xrefs.yaml:2907:7 [/265/entity_types/3] key 'type_id:' is required.
metadata/db-xrefs.yaml:2907:7 [/265/entity_types/3/example_id] '37232': not a string.
metadata/db-xrefs.yaml:2908:7 [/265/entity_types/4] key 'type_id:' is required.
metadata/db-xrefs.yaml:2908:7 [/265/entity_types/4] key 'type_name:' is required.
metadata/db-xrefs.yaml#0: INVALID
The command "kwalify -E -f metadata/db-xrefs.schema.yaml metadata/db-xrefs.yaml 2>&1 | tee out.log && grep 'INVALID\|ERROR' out.log; test $? -ne 0" exited with 1.
```
You might want to take a look at YAML and the schema: https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.schema.yaml

**Comment by zhilianghu** [comment]
Updated with generic URL, type_id, type_name, and example_ID.

**Comment by kltm** [comment]
@zhilianghu I believe that these should not have brackets:
` - database: [Animal_CorrDB]`
As well, the tests are not passing at these points:
https://travis-ci.org/github/geneontology/go-site/builds/751254307#L1095
https://travis-ci.org/github/geneontology/go-site/builds/751254307#L1107
generic_urls are a YAML list. You can either look at the schema (https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.schema.yaml) or other examples in the file.

**Comment by zhilianghu** [comment]
Thank you for your patience with me.  New changes are made.

**Comment by kltm** [comment]
@zhilianghu You may want to look at the schema or one of the other entries in the file that is close to yours in content and mimic it--it appears that your are confusing list and "dictionary" structures in YAML, with the overuse of `-`.
https://travis-ci.org/github/geneontology/go-site/builds/753374096#L1093

As you work on your PR, you can see the current status under "Details" after "All checks have failed". The part that you'd be interested in would be after:
```
kwalify -E -f metadata/db-xrefs.schema.yaml metadata/db-xrefs.yaml 2>&1 | tee out.log && grep 'INVALID\|ERROR' out.log; test $? -ne 0
```
You can see the errors after that.

**Comment by kltm** [comment]
This looks clean to me.

@dougli1sqrd Would you mind taking a quick look at this too to make sure nothing looks off?

**Comment by dougli1sqrd** [comment]
This seems good!

**Pull request merged**